### PR TITLE
Damans227/adding map

### PR DESCRIPTION
**Pull request by Damans227**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

This pull request is a work in progress to add the following enhancement: https://github.com/MLH-Fellowship/pod-4.1.5-portfolio/issues/10

Test server: [add-map-prep-pod-4-1-5.netlify.app](https://add-map-prep-pod-4-1-5.netlify.app/)

Resources leveraged to build the custom map: 

1) [Leaflet](https://leafletjs.com/)
2) [Tutorial](https://www.youtube.com/watch?v=nZaZ2dB6pow)

### CODE REVIEW

**Comment by Damans227** [comment]
@Shikhar15606 @khattakdev Ptal, thx!

**Comment by khattakdev** [comment]
@Damans227 Always add someone as a reviewer to check and review your code.

**Comment by Damans227** [comment]
> @Damans227 Always add someone as a reviewer to check and review your code.

Ok.

**Review by P0intMaN — APPROVED** [review]
LGTM! As far this PR is concerned, it doesn't deal with the 'pop-up card feature' right? 

By pop-up card feature, I mean when a user clicks on the profile, a card would pop up displaying the basic info (role, country).

**Comment by P0intMaN** [comment]
I mean, that can be a part of another PR. This PR satisfies the purpose it was created for, Awesome work @Damans227 @Shikhar15606

**Comment by Damans227** [comment]
> LGTM! As far this PR is concerned, it doesn't deal with the 'pop-up card feature' right? 
> 
> By pop-up card feature, I mean when a user clicks on the profile, a card would pop up displaying the basic info (role, country).

Yea, it's only half baked is what I would say. Adding user details and possibly making it a pop up card is what we are aiming to work on as next steps in this PR.

**Comment by Damans227** [comment]
> That can be a part of another PR though. This PR satisfies the purpose it was created for, Awesome work @Damans227 @Shikhar15606 

Much thx!

**Review by Shikhar15606 — CHANGES_REQUESTED** [review]
Great Work @Damans227. We will need to make a few changes.

File: _sass/map.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#map {
+  height: 500px;
+  width: 800px;
```
[the remark this answers is not in the document]
Comment (Shikhar15606) [reply _sass/map.scss:3]: Fixed it

File: assets/js/marker.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  iconUrl: "/assets/img/daman.png",
+  iconSize: [50, 50], // size of the icon
+  iconAnchor: [22, 94], // point of the icon which will correspond to marker's location
+  popupAnchor: [-3, -76], // point from which the popup should open relative to the iconAnchor
+  className: "markerImage",
+});
+
+imageOfPodMembers.map((name) => {
+  (markerIcon.iconUrl = `/assets/img/${name}.png`),
+    L.marker([45.4214, -75.6901], { icon: markerIcon }).addTo(map);
```
[the remark this answers is not in the document]
Comment (Shikhar15606) [reply assets/js/marker.js:15]: Fixed it

File: _includes/podmap.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="map-container">
```
[the remark this answers is not in the document]
Comment (Shikhar15606) [reply _includes/podmap.html:1]: Fixed it

File: _includes/header.html [hunk 4]
```diff
@@ -1,32 +1,64 @@
 <head>
-    <!-- Required meta tags -->
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta property="og:title" content="Personal Portfolio">
-    <meta property="og:description"
-        content="My Personal Portfolio">
-    <meta property="og:image" content="./assets/img/social-link.jpg">
-    <meta property="og:url" content="{{ page.url }}">
+  <!-- Required meta tags -->
```
[the remark this answers is not in the document]
Comment (Damans227) [reply _includes/header.html:2]: Thanks for the feedback.

File: _includes/header.html [hunk 5]
```diff
@@ -1,32 +1,64 @@
...
-        crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js"
-        integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut"
-        crossorigin="anonymous"></script>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js"
-        integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k"
-        crossorigin="anonymous"></script>
-    <script src="https://jsxss.com/assets/xss.js"
-        crossorigin="anonymous"></script>
+  <script
```
[the remark this answers is not in the document]
Comment (Damans227) [reply _includes/header.html:13]: Yup, I will remember switching prettier off next time.

File: _includes/header.html [hunk 6]
```diff
@@ -1,32 +1,64 @@
...
 
-    <link lang='sass' rel="stylesheet" href="/assets/css/main.css">
-    <link rel='icon' href='/assets/img/favicon.ico' type='image/x-icon' />
-    <title>{{ site.title }}</title>
-</head>
\ No newline at end of file
+  <link lang="sass" rel="stylesheet" href="/assets/css/main.css" />
+  <link rel="icon" href="/assets/img/favicon.ico" type="image/x-icon" />
+
+  <!-- Leaflet CSS and Script-->
```
[the remark this answers is not in the document]
Comment (Damans227) [reply _includes/header.html:50]: Totally!

File: _includes/podmap.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (Damans227) [reply _includes/podmap.html:1]: Thanks Shikhar!

**Comment by khattakdev** [comment]
should I merge this PR @P0intMaN @Shikhar15606 ?

**Review by esh04 — APPROVED** [review]
Looks good to me!

**Review by khattakdev — APPROVED** [review]

**Pull request merged**